test/e2e/metrics_util.go

```diff
@@ -140,6 +143,11 @@ func HighLatencyRequests(c *client.Client, threshold time.Duration) (int, error)
 	badMetrics := 0
 	top := 5
 	for _, metric := range metrics.APICalls {
+		threshold := 250 * time.Millisecond
```

Comment:
Can you move thresholds to constants in this file? We'll probably need to tweak them when we'll increase the number of supported nodes or max-pods.